R/git.R

```diff
@@ -323,85 +323,117 @@ git_clean <- function() {
 #' Git/GitHub sitrep
 #'
 #' Get a situation report on your current Git/GitHub status. Useful for
-#' diagnosing problems.
+#' diagnosing problems. The default is to report all values; provide values
+#' for `tool` or `scope` to be more specific.
+#'
+#' @param tool which tool(s) to report: one or more of `"git"`, `"github"`.
+#' @param scope which scope(s) to report: one or more of `"global"`, `"project`.
+#'
 #' @export
 #' @examples
 #' \dontrun{
+#' # report all
 #' git_sitrep()
+#'
+#' # report git global
+#' git_sitrep("git", "global")
 #' }
-git_sitrep <- function() {
+git_sitrep <- function(tool = c("git", "github"),
+                       scope = c("global", "project")) {
```

Comment:
The signature of `use_git_config(scope = c("user", "project"), ...)` uses `"user"`, not `"global"`. Ditto for `edit_git_config()` and `edit_git_ignore()` (and other non-git/github-y functions, which is presumably the rationale). I think we should do the same here, for consistency. And then you can borrow similar phrasing to document the `scope` parameter:

```
#' @param scope Report globally for the current __user__, or locally for the
#'   current __project__
```

Reply:
That makes sense - I have updated the signature and documention.